CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.1.12
+
+* Rename cotsoc_noncontrib to cotisations_non_contributives
```

Comment:
There are only `cotsoc_bar_declarant1` and `cotsoc_lib_declarant1` remaining. How should I rename them?